src/Telemetry/Exit_Interview/Exit_Interview_Subscriber.php

```diff
@@ -80,24 +80,24 @@ public function render_exit_interview() {
 	 * @return void
 	 */
 	public function ajax_exit_interview() {
-		$uninstall_reason_id = filter_input( INPUT_POST, 'uninstall_reason_id', FILTER_SANITIZE_STRING );
+		$uninstall_reason_id = filter_input( INPUT_POST, 'uninstall_reason_id', FILTER_SANITIZE_SPECIAL_CHARS );
```

Comment:
Should this be using [htmlspecialchars()](https://www.php.net/manual/en/function.htmlspecialchars.php) instead? 

As suggested by the docs: 
<img width="966" alt="image" src="https://github.com/stellarwp/telemetry/assets/6947218/26ca59eb-ecdb-4642-b611-905f7c1b2fc9">

Reply:
I was a little stumped by this, the official line is to use `htmlspecialchars()` when outputting the data and `FILTER_DEFAULT` when using `filter_input` - but the PHPCS doesn't allow `FILTER_DEFAULT` here.

Should I use `htmlspecialchars()` when saving anyway?

Reply:
Digging in a bit more, I think `FILTER_SANITIZE_SPECIAL_CHARS` is fine. As long as you've tested this, I'm good with it 👍🏼